man/tpm2_ecdhzgen.1.md

```diff
@@ -0,0 +1,63 @@
+% tpm2_ecdhzgen(1) tpm2-tools | General Commands Manual
+
+# NAME
+
+**tpm2_ecdhzgen**(1) - Recovers the shared secret value (Z) from a public point
+and a specified private key.
+
+# SYNOPSIS
+
+**tpm2_ecdhzgen** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_ecdhzgen**(1) - Recovers the shared secret value (Z) from a public point
+and a specified private key. It will perform the multiplication of the provided
+inPoint (QB) with the private key (ds) and return the coordinates of the
+resultant point (Z = (xZ , yZ) ≔ [hds]QB; where h is the cofactor of the curve).
+
+# OPTIONS
+
+  * **-c**, **\--key-context**=_FILE_:
+
+    Context object pointing to ECC key.
+    Either a file or a handle number. See section "Context Object Format".
+
+  * **-p**, **\--key-auth**=_AUTH_:
+
+    The authorization value for the ECC key object.
+
+  * **-u**, **\--public**=_FILE_:
+
+    Output ECC point Q
```

Comment:
Needs a period at the end of the sentence. Q is what you share with the other party during the ECDH exchange right?

Reply:
correct